satellite/console/consoleweb/server.go

```diff
@@ -112,31 +121,42 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, mail
 		server.config.ExternalAddress = "http://" + server.listener.Addr().String() + "/"
 	}
 
-	mux := http.NewServeMux()
 	fs := http.FileServer(http.Dir(server.config.StaticDir))
 
-	mux.Handle("/api/graphql/v0", http.HandlerFunc(server.grapqlHandler))
+	router := mux.NewRouter()
+	router.Handle("/api/graphql/v0", http.HandlerFunc(server.grapqlHandler))
+
+	usersRouter := router.PathPrefix("/users").Subrouter()
+	usersRouter.Use(server.authMiddlewareHandler)
+
+	usersRouter.Handle("/", http.HandlerFunc(server.createNewUserRequestHandler)).Methods("POST")
+	usersRouter.Handle("/", http.HandlerFunc(server.deleteAccountRequestHandler)).Methods("DELETE")
```

Comment:
I've found that `deleteAccountRequestHandler` is receiving some information in the request body, however, according to the RFC (https://tools.ietf.org/html/rfc7231#section-4.3.5) the `DELETE` method request message has not a defined semantic, hence I would change this method by a `POST`.